### ISSUE TITLE
[NUI] AccessibilityCalculateStatesWrapper Missing null check

### ISSUE THREAD

**Issue by myroot** [Samsung/TizenFX#4369]
https://github.com/Samsung/TizenFX/blob/356b2ee819f33e19f15e668868339feebd086d98/src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityWrappers.cs#L95-L109

view could be null, but didn't check null
